sync/sync_test.go

```diff
@@ -57,8 +57,8 @@ func TestSyncBlocks(t *testing.T) {
 		b0, err := gw.BlockByNumber(context.Background(), 0)
 		require.NoError(t, err)
 		s0, err := gw.StateUpdate(context.Background(), 0)
-		require.NoError(t, err)
-		require.NoError(t, bc.Store(b0, s0))
+		assert.NoError(t, err)
```

Comment:
This line should still be `require`. Otherwise, the next line would panic if s0 is nil.